susemanager-utils/susemanager-sls/salt/certs/Amazon2.sls

```diff
@@ -0,0 +1 @@
+RedHat7.sls
```

Comment:
I think you need to create a new Amazon2.sls file here if you want to install the Amazon Linux 2 repo key. See https://github.com/paususe/uyuni/blob/492f64fca41f1292444690e8b42e16a812b28970/susemanager-utils/susemanager-sls/salt/certs/Alibaba2.sls

Reply:
Not really. We don't need the Amazon Linux 2 repo key. That's trusted by default by the instance. I am not wrong this RedHat7.sls is the RES7 client tools GPG key. What we should have is this or the correct GPG Key for the Uyuni client tools, with the links adjusted by the SPEC depending on the base OS used to build susemanager-sls. 

But I will not do this work on this PR to avoid making it more complicate, and also because it should affect **all** OS.